features/step-definitions/task.2.js

```diff
@@ -0,0 +1,144 @@
+
+const { Given, When, Then } = require('@cucumber/cucumber');
+const HomePage = require('../pageobjects/home.page');
+const HoverPage = require('../pageobjects/hover.page');
+const AlertPage=require('../pageobjects/alert.page');
+const send_text='sadia'
+const image_number=2;
+
+//test case 3
+
+Given(/^I am on the Home page$/,async () => {
+	await HomePage.open()
+});
+
+When(/^I click the hovers link$/,async() => {
+    await HomePage.ClickHoverLink()
+});
+
+Then(/^I am on the hovers page$/, async() => {
+	await expect(HoverPage.locate_User_Image(image_number)).toBeExisting()
+   
+});
+
+When(/^I Hover over the profile image of user2$/, async() => {
+    
+    await HoverPage.locate_User_Image(image_number).moveTo();
+   
+});
+
+Then(/^I see name: user 2 and view profile link are displayed under the image on hover$/, async() => {
+    await expect(HoverPage.locate_User_Profile(image_number)).toBeDisplayed()
+	
+});
+
+
+
+
+//test case 4
+
+Given(/^I am on the Home page$/, async() => {
+	await HomePage.open()
+});
+
+When(/^I click on the key press link$/, async() => {
+	
+    await HomePage.ClickKeyPageLink()
+});
+
+Then(/^I am on the Key Press page$/, async() => {
+	
+    await expect(keyPage.TextField).toBeExisting()
+});
+
+When(/^I press Backspace key$/, async() => {
+    
+    await browser.keys(['Backspace'])
+});
+
+Then(/^I can see the message You entered: BACK_SPACE$/,async () => {
+	
+    await expect(keyPage.ResultField).toHaveTextContaining('You entered: BACK_SPACE')
+});
+
+
+
+//test case 5
+
+
+Given(/^I am on the Home page$/, async() => {
+	await HomePage.open()
+});
+
+When(/^I click on the JavaScript Alerts link$/, async() => {
+    await HomePage.ClickJSAlertLink()
+});
+
+Then(/^I am on the JavaScript Alerts page$/, async() => {
+	
+    await expect(AlertPage.JSAlert).toBeExisting()
+});
+
+When(/^I press Click for JS Alert button$/, async() => {
+	await AlertPage.ClickJSAlert()
+});
+
+Then(/^I can see the Alert$/, async() => {
+    
+	let alertText = await browser.getAlertText() ;
+    expect(alertText).toEqual('I am a JS Alert');
+});
+
+When(/^I close the alert$/, async() => {
+	browser.acceptAlert()
+});
+
+Then(/^I can see the alert is closed$/, async() => {

```

Comment:
would rename it to 
```suggestion
Then(/^I can see the user prompt is closed$/, async() => {

```
to make it more reusable